chunk/manager.go

```diff
@@ -13,6 +13,7 @@ type Manager struct {
 	downloader *Downloader
 	storage    *Storage
 	queue      chan *QueueEntry
+	acknowledgeAbuse bool
```

Comment:
Please fix indentation of the struct definition.

Reply:
well that's embarrassing.  287938feea0b1246bea1977dc59e9f2ef003c4d6 runs `gofmt` on `manager.go` and `download.go`